core/src/main/java/org/wso2/testgrid/core/command/GenerateTestPlanCommand.java

```diff
@@ -630,4 +628,20 @@ protected NodeTuple representJavaBeanProperty(Object javaBean, Property property
             }
         }
     }
+
+    /**
+     * If the testgrid yaml file is hidden in the directory, change the URI as to refer the hidden file.
+     * @param directory directory where the testgrid yaml file exists
+     * @return testgrid yaml file path
+     */
+    private Path getTestGridYamlLocation(String directory) {
+        String hiddenFileDot = ".";
+        Path yamlPathIfHidden = Paths.get(directory, hiddenFileDot + TestGridConstants.TESTGRID_YAML);
+        Path yamlPathIfNotHidden = Paths.get(directory, TestGridConstants.TESTGRID_YAML);
```

Comment:
Shall we modify these variable names as following way 
yamlPathIfHidden --> hiddenYamlPath
yamlPathIfNotHidden ---> defaultYamlPath
WDYT?